component/azstorage/utils.go

```diff
@@ -553,6 +553,14 @@ func split(prefixPath string, path string) string {
 	return filepath.Join(paths...)
 }
 
+// removePrefixPath removes the given prefixPath from the beginning of path,
+// if it exists, and returns the resulting string without leading slashes.
+func removePrefixPath(prefixPath, path string) string {
+	prefixPath = strings.TrimLeft(prefixPath, "/")
```

Comment:
This just removes the "/" from prefixPath. Do we really need to call TrimLeft or just checking the first character will be faster here?